pkg/test/helpers.go

```diff
@@ -170,6 +174,26 @@ func DeleteStorageClass(ctx context.Context, c client.Client) {
 	}
 }
 
+func CreateFakeTLSSecret(ctx context.Context, vdb *vapi.VerticaDB, c client.Client, name string) {
+	secret := BuildTLSSecret(vdb, name, TestKeyValue, TestCertValue, TestCaCertValue)
```

Comment:
Could we combine this function with `createTLSSecret` in install_reconciler_test.go? There is a lot of repetition.